python_modules/dagster/dagster/_core/definitions/partition_mapping.py

```diff
@@ -292,6 +304,9 @@ def get_downstream_partitions_for_partitions(
             )
         return downstream_partitions_def.empty_subset()
 
+    def __str__(self) -> str:
+        return f"Each downstream partition depends on the following upstream partitions: {self.partition_keys}"
```

Comment:
In some cases, this could be a large number of partitions. Should we be concerned?

Reply:
It is a little concerning, but for StaticPartitionsDefinitions our description string is all of the keys concatenated with a comma, which is probably worse

Reply:
We could just truncate and add an ellipsis on the front end if the string is too long?

Reply:
Got it. That sounds fine to me for now.